airflow/ui/src/pages/Providers.tsx

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Heading, Link } from "@chakra-ui/react";
+import type { ColumnDef } from "@tanstack/react-table";
+
+import { useProviderServiceGetProviders } from "openapi/queries";
+import type { ProviderResponse } from "openapi/requests/types.gen";
+import { DataTable } from "src/components/DataTable";
+import { ErrorAlert } from "src/components/ErrorAlert";
+
+const columns: Array<ColumnDef<ProviderResponse>> = [
+  {
+    accessorKey: "package_name",
+    cell: ({ row: { original } }) => (
+      <Link
+        aria-label={original.package_name}
+        color="fg.info"
+        href={`https://airflow.apache.org/docs/${original.package_name}/${original.version}/`}
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        {original.package_name}
+      </Link>
+    ),
+    header: "Package Name",
+  },
+  {
+    accessorKey: "version",
+    cell: ({ row: { original } }) => original.version,
+    header: () => "Version",
+  },
+  {
+    accessorKey: "description",
+    cell: ({ row: { original } }) => {
+      const urlRegex = /http(s)?:\/\/[\w.-]+(\.?:[\w.-]+)*([#/?][\w!#$%&'()*+,./:;=?@[\]~\-]*)?/g;
```

Comment:
I wasn't aware that our eslint rules was enforcing a `u` flag. I guess we need to keep it then 👍